cloudman/helmsman/clients/kubectl_client.py

```diff
@@ -0,0 +1,57 @@
+"""A wrapper around the kubectl commandline client"""
+import shutil
+from . import helpers
+
+
+class KubeCtlService(object):
+    """Marker interface for CloudMan services"""
+    def __init__(self, client):
+        self._client = client
+
+    def client(self):
+        return self._client
+
+
+class KubeCtlClient(KubeCtlService):
```

Comment:
This may be better of renamed to KubernetesClient because in principle, the fact that we are wrapping kubectl is not relevant to the client? We could just as well swap the implementation to use pykube and it should still work the same?

Reply:
Done in https://github.com/galaxyproject/cloudman/pull/105/commits/49e82f3c44917d54f3a3730bc5dc6ca2a785ab1f